overlord/devicestate/firstboot.go

```diff
@@ -39,7 +39,7 @@ import (
 
 var errNothingToDo = errors.New("nothing to do")
 
-func installSeedSnap(st *state.State, sn *seed.Snap, flags snapstate.Flags) (*state.TaskSet, *snap.Info, error) {
+func installSeedSnap(model *asserts.Model, st *state.State, sn *seed.Snap, flags snapstate.Flags) (*state.TaskSet, *snap.Info, error) {
```

Comment:
given that this takes flags from outside, can't we set this extra flag from outside as well?

Reply:
yes I suppose it could, that's fair I just thought this seemed like a fair place to put it since we call installSeedSnap from multiple places